File: src/002. Even Fibonacci numbers/002.c
#include <stdio.h>

int main(void) {
    long long f1 = 1, f2 = 1;
    long long sum = 0;
    while (f1 < 4000000LL) {
        long long temp = f2;
        f2 = f1 + f2;
        f1 = temp;
        if (f1 % 2 == 0) {
            sum += f1;
        }
    }
    printf("%lld\n", sum);
    return 0;
}

File: src/002. Even Fibonacci numbers/002.java
public class Main {
    public static void main(String args[]) {
        long f1 = 1;
        long f2 = 2;
        
        long sum = 0;
        while (f1 < 4_000_000) {
            if (f1 % 2 == 0) {
                sum += f1;
            }
            long temp = f2;
            f2 = f1 + f2;
            f1 = temp;
        }
        System.out.println(sum);
    }
}
